config/initializers/secrets.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# Be sure to restart your server after modifying this file.
+
+require "pathname"
+require "velum/secrets"
+
+env_path = Pathname.new("#{ENV.fetch("VELUM_SECRETS_DIR")}/key_base.json")
+Rails.application.secrets.secret_key_base = Velum.create_secret_key_base(env_path)
```

Comment:
Maybe I'd rename `create_secret_key_base` to `read_or_create_secret_key_base`. Just to give it a name closer to what it does.